app/constants/task-type.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   CALCULATE_WORKLOAD_POINTS: 'CALCULATE-WORKLOAD-POINTS',
   CREATE_WORKLOAD: 'CREATE-WORKLOAD',
-  PROCESS_IMPORT: 'PROCESS-IMPORT',
-  CREATE_WORKLOAD_REPORT: 'CREATE-WORKLOAD-REPORT'
```

Comment:
I think so, there is no specific worker / task for CREATE-WORKLOAD-REPORT. They are inserted as part of PROCESS-IMPORT